### PR TITLE
LPS-153299 Enable row highlight on hover for FDS table view

### PR DESCRIPTION
**Pull request by beltranrengifo**
[Jira task](https://issues.liferay.com/browse/LPS-153299)

### Motivation 

On mouse hover event, the table row should be highlighted.

### How to 

Just deploy and place FDS Sample module in any page

image

### CODE REVIEW

**Review by markocikos — COMMENTED** [review]
@beltranrengifo See comment inline.

As far as I'm concerned, the blue on hover looks much better then current UI, and consistent with [Lexicon guidelines on tables](https://liferay.design/lexicon/core-components/table/#hover-state). I think we can push to apply to DXP-wide, and rollback in the unlikely case of a backlash. cc @dsanz @drakonux

File: modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/styles/_dnd_table.scss [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 .dnd-tbody {
 	display: table-row-group;
+
+	&.with-hover {
+		.dnd-tr:hover {
+			background-color: $primary-l3;
+		}
+	}
```
[the remark this answers is not in the document]
Comment (beltranrengifo) [reply modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/styles/_dnd_table.scss:12]: Yes, you're right. I saw that pattern using `with-` and I tried to give it some continuity but I think your suggestion is more clear. I'll check it.

**Comment by beltranrengifo** [comment]
ci:test:sf

**Comment by beltranrengifo** [comment]
ci:test:relevant

**Comment by beltranrengifo** [comment]
ci:test:relevant

**Comment by beltranrengifo** [comment]
> ### ✔️ ci:test:stable - 26 out of 26 jobs passed
> ### ❌ ci:test:relevant - 64 out of 70 jobs passed in 1 hour 16 minutes
> Click here for more details.
> #### Base Branch:
> Branch Name: [master](https://github.com/liferay/liferay-portal/tree/master)Branch GIT ID: [9f5b07fc4d34a7ac4959d68ae3a53e7b29db561a](https://github.com/liferay/liferay-portal/commit/9f5b07fc4d34a7ac4959d68ae3a53e7b29db561a)
> 
> #### Upstream Comparison:
> Branch GIT ID: [9f5b07fc4d34a7ac4959d68ae3a53e7b29db561a](https://github.com/liferay/liferay-portal/commit/9f5b07fc4d34a7ac4959d68ae3a53e7b29db561a)Jenkins Build URL: [Acceptance Upstream DXP (master) #3261](https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//3261)
> 
> **ci:test:stable - 26 out of 26 jobs PASSED**
> **ci:test:relevant - 64 out of 70 jobs PASSED**
> ##### For more details click [here](https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3305/jenkins-report.html).
> #### Failures unique to this pull:
> 1. [functional-tomcat90-mysql57-jdk8/1/6](https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/5345//consoleText)1. [LocalFile.FrontendDataSet#ActionsCanBeCustomized](https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/5345//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_FrontendDataSet_ActionsCanBeCustomized_) - [Poshi Report](https://testray.liferay.com/reports/production/logs/2022-05/test-1-30/test-portal-acceptance-pullrequest(master)/3305/functional-tomcat90-mysql57-jdk8/1/6/LocalFile.FrontendDataSet_ActionsCanBeCustomized/index.html.gz) - [Poshi Summary](https://testray.liferay.com/reports/production/logs/2022-05/test-1-30/test-portal-acceptance-pullrequest(master)/3305/functional-tomcat90-mysql57-jdk8/1/6/LocalFile.FrontendDataSet_ActionsCanBeCustomized/summary.html.gz) - [Console Output](https://testray.liferay.com/reports/production/logs/2022-05/test-1-30/test-portal-acceptance-pullrequest(master)/3305/functional-tomcat90-mysql57-jdk8/1/6/jenkins-console.txt.gz)
> 
> **Failures in common with [acceptance upstream results](https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)) at [9f5b07f](https://github.com/liferay-frontend/liferay-portal/commit/9f5b07fc4d34a7ac4959d68ae3a53e7b29db561a):**
> ##### Test bundle downloads:
> * [liferay-portal-bundle-tomcat.tar.gz](https://files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/3305/liferay-portal-bundle-tomcat.tar.gz) (mirrors: [dlc](http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/3305/liferay-portal-bundle-tomcat.tar.gz), [lax](http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/3305/liferay-portal-bundle-tomcat.tar.gz))

This test did not fail yesterday but today it is :(

**Comment by beltranrengifo** [comment]
ci:forward

**Pull request closed without merging**